draft-ietf-httpbis-priority.md

```diff
@@ -369,70 +383,82 @@ Stream ID:
 Priority Field Value:
 : The priority update value in ASCII text, encoded using Structured Headers.
 
-The HTTP/2 PRIORITY_UPDATE frame MUST NOT be sent prior to opening the
-stream.  If a PRIORITY_UPDATE is received prior to the stream being opened,
-it MAY be treated as a connection error of type PROTOCOL_ERROR.
+The PRIORITY_UPDATE frame MAY be sent before the stream that it references has
+been created. The Stream ID MUST be within the stream limit. If a server
+receives a PRIORITY_UPDATE for a Stream ID that is beyond the stream limits,
+this SHOULD be treated as a connection error of type PROTOCOL_ERROR.
+PRIORITY_UPDATE frames received before the request or response has started
+SHOULD be buffered until the stream is opened and applied immediately after the
+request message has been processed. Holding PRIORITY_UPDATES consumes extra
+state on the peer, although the size of the state is bounded by stream limits.
+There is no bound on the number of PRIORITY_UPDATEs that can be sent, so an
+endpoint SHOULD store only the most recently received frame.
+
+If a PRIORITY_UPDATE frame is received with a Prioritized Stream ID of 0x0, the
+recipient MUST respond with a connection error of type PROTOCOL_ERROR.
 
-TODO: add more description of how to handle things like receiving
-PRIORITY_UPDATE on wrong stream, a PRIORITY_UPDATE with an invalid ID, etc.
+If a client receives a PRIORITY_UPDATE frame, it MUST respond with a connection
+error of type PROTOCOL_ERROR.
 
 ## HTTP/3 PRIORITY_UPDATE Frame {#h3-update-frame}
 
-The HTTP/3 PRIORITY_UPDATE frame (type=0xF) carries the identifier of the
-element that is being reprioritized, and the updated priority in ASCII text,
-using the same representation as that of the Priority header field value.
+The HTTP/3 PRIORITY_UPDATE frame (type=0x1CCB8BBF1F0700 or 0x1CCB8BBF1F0701) is
```

Comment:
Is it 0x1CCB8BBF1F0700/0x1CCB8BBF1F0701 or 0x10/0x11?

Reply:
In HTTP/3, the value should be the long 0x1CCB8BBF1F0700/0x1CCB8BBF1F0701, I'll fix this up.